services/assets/info.go

```diff
@@ -48,7 +48,7 @@ func (cl *HttpAssetClient) GetCoinInfo(coinId int, token string) (*watchmarket.C
 
 	err = json.Unmarshal(resp.Body(), &info)
 	if err != nil {
-		return nil, err
+		return &info, errors.New(fmt.Sprintf("Unmarshal %s", err.Error()))
```

Comment:
minor/nonblocking: "Failed to unmarshal" would be more descriptive.

Reply:
Ok